knip.production.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+	"$schema": "https://unpkg.com/knip@next/schema.json",
+	"entry": ["src/**/index.ts!"],
+	"project": ["src/**/*.ts!", "!src/test/**/*.ts!", "!src/**/*.test.ts!"]
+}
```

Comment:
@webpro It doesn't seem to currently be possible to use a single config file for our use case.

Essentially, we want this config, but with the comment addressed.

```jsonc
{
	"$schema": "https://unpkg.com/knip@next/schema.json",
	"entry": ["src/**/index.ts!", "src/**/*.test.ts"],
	"project": [
		"src/**/*.ts!",
		"!src/test/**/*.ts!", // <-- We want these "not" globs to only apply in production mode.
		"!src/**/*.test.ts!"  // <-- Using a separate production specific config seems to be the only way to currently achieve this.
	]
}
```

When using config that runs logic (e.g. `knip.ts`), it would be great if we could access a `isProduction` variable.

Reply:
Sorry for the late reply, but somehow today this popped up in the back of my mind. Any chance you could give it a shot with the latest Knip v1? I think you don't need **any** configuration anymore for either dev or production mode :)